src/mc_rbdyn/configuration_io.cpp

```diff
@@ -685,15 +685,15 @@ rbd::parsers::Geometry::Mesh ConfigurationLoader<rbd::parsers::Geometry::Mesh>::
 {
   rbd::parsers::Geometry::Mesh m;
   m.filename = static_cast<std::string>(config("filename"));
-  m.scale = config("scale");
+  m.scaleV = config("scaleV");
```

Comment:
To keep it compatible if people update mc_rtc without updating the GUI implementations
```suggestion
  if(auto scale = config.find("scale"))
  {
    m.scale = scale->operator<double>();
    m.scaleV.setConstant(m.scale);
  }
  else
  {
    m.scaleV = config("scaleV");
    m.scale = m.scaleV(0);
  }
```

Reply:
`m.scale` won't exist after the RBDyn PR, so I'm not sure what you are suggesting here?

Reply:
Oh yes. I guess we can skip this then release a new version of mc_rtc that does not require `scale` and then release a version of `RBDyn` without scale?

Reply:
Yes as is it works without updating `RBDyn`, it's just less confusing not to have both properties in the future ;)